Source/CarthageKit/Xcode.swift

```diff
@@ -355,6 +357,55 @@ private func mergeExecutables(_ executableURLs: [URL], _ outputURL: URL) -> Sign
 		.then(SignalProducer<(), CarthageError>.empty)
 }
 
+private func mergeSwiftHeaderFiles(_ executableURLs: [URL], _ executableOutputURL: URL) -> SignalProducer<(), CarthageError> {
+	precondition(executableOutputURL.isFileURL)
+	
+	let conditionalsCheck = "#if 0"
+	let conditionalPrefix = "#if 0\n#elif (defined(__x86_64__) && __x86_64__) || (defined(__i386__) && __i386__)\n"
+	let conditionalSuffix = "\n#endif"
+	
+	guard let conditionalsCheckContents = conditionalsCheck.data(using: .utf8) else { return .empty }
+	guard let conditionalPrefixContents = conditionalPrefix.data(using: .utf8) else { return .empty }
+	guard let conditionalSuffixContents = conditionalSuffix.data(using: .utf8) else { return .empty }
```

Comment:
I think these should just use `!`. Since the contents are static, this absolutely a programmer error that we want to catch. Crashing will make it obvious; returning an empty producer will hide the error.

Reply:
Good point, will do.